README.md

```diff
@@ -20,30 +20,33 @@ AutoTSS is a Discord bot that automatically saves [SHSH blobs](https://www.theip
 | Saves SHSH blobs for beta iOS versions | ✅ | ✅ | ❌ | ✅*** | ✅*** | ❌ |
 | Detects signing parties | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ |
 
-<sup>* Requires users to already have a generator-apnonce combination for their device, more information on that can be found [here](https://www.reddit.com/r/jailbreak/comments/m3744k/tutorial_shsh_generatorbootnonce_apnonce_nonce/).</sup>
+<sup>* Requires users to already have a generator-apnonce combination for their device, more information on that can be found [here](https://www.reddit.com/r/jailbreak/comments/m3744k/tutorial_shsh_generatorbootnonce_apnonce_nonce/).</sup><br>
+<sup>** Requires users to leave blobsaver always running.</sup><br>
+<sup>*** Requires users to manually specify a BuildManifest from a beta IPSW.</sup>
 
-<sup>** Requires users to leave blobsaver always running.</sup>
+## Running
 
-<sup>*** Requires users to manually specify a BuildManifest from a beta IPSW.</sup>
+While you can run your own instances, it's prefered for you to simply use the already hosted instance. Support for your own instance will not be provided. Windows is not supported!
 
-## Setup
-To locally host, follow these steps:
+To locally host your own instance, [create a Discord bot](https://discord.com/developers) and follow these steps...
 
-0. Create a [Discord Bot](https://discord.com/developers/applications).
-    - Under the bot menu, make sure to enable the 'Server Members Intent'.
+1. Build and in install [`tsschecker`](https://github.com/DanTheMann15/tsschecker) and its dependencies
```

Comment:
Typo, also change to [1Conan's fork](https://github.com/1Conan/tsschecker) of tsschecker.